### PR TITLE
feat: add link to folders support page

### PR DESCRIPTION
**Pull request by thisisamir98**
## Description
Adds a link to support page for folders on sidebar

## Screenshots/Screencast (for UI changes)
image

### CODE REVIEW

File: src/script/page/LeftSidebar/panels/Conversations/ConversationTab/ConversationFolderTab.tsx [hunk 1]
```diff
@@ -92,12 +96,17 @@ export const ConversationFolderTab = ({
...
-        {unreadConversations.length > 0 && (
-          <span className="conversations-sidebar-btn--badge">{unreadConversations.length}</span>
-        )}
       </button>
       <div className={cx('conversations-sidebar-folders', {active: isFoldersTabOpen})}>
         <div className="conversations-sidebar-folders--inner-wrapper">
+          {folders.length === 0 && (
+            <div
+              className="conversations-sidebar-folders--empty"
+              dangerouslySetInnerHTML={{
```
Comment (aweiss-dev) [inline src/script/page/LeftSidebar/panels/Conversations/ConversationTab/ConversationFolderTab.tsx]: Always feel a cold shiver when I see this in the code

**Review by aweiss-dev — APPROVED** [review]

**Review by atomrc — CHANGES_REQUESTED** [review]
Please use the `ReactLocalizerUtil` to translate string and transforms content into react components (thus avoiding the need for `dangerouslySetInnerHTML`)

File: src/script/page/LeftSidebar/panels/Conversations/ConversationTab/ConversationFolderTab.tsx (hunk 1 above)
Comment (thisisamir98) [reply src/script/page/LeftSidebar/panels/Conversations/ConversationTab/ConversationFolderTab.tsx]: https://github.com/wireapp/wire-webapp/pull/17214/commits/5eab1f5c658f4522e2195ce8dcc5e13f1174a396

File: src/script/page/LeftSidebar/panels/Conversations/ConversationTab/ConversationFolderTab.tsx [hunk 2]
```diff
@@ -92,12 +94,17 @@ export const ConversationFolderTab = ({
...
-        {unreadConversations.length > 0 && (
-          <span className="conversations-sidebar-btn--badge">{unreadConversations.length}</span>
-        )}
       </button>
       <div className={cx('conversations-sidebar-folders', {active: isFoldersTabOpen})}>
         <div className="conversations-sidebar-folders--inner-wrapper">
+          {folders.length === 0 && (
+            <div className="conversations-sidebar-folders--empty">
+              {t('conversationFoldersEmptyText')}
+              <a href={Config.getConfig().URL.SUPPORT.FOLDERS} target="_blank" rel="noreferrer">
```
Comment (przemvs) [inline src/script/page/LeftSidebar/panels/Conversations/ConversationTab/ConversationFolderTab.tsx:110]: You should use Link from react-ui-kit there :) Check another usages

**Pull request merged**